CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Breaking Changes
 ### Added
+- Added `backgroundColor` `ex.Engine` constructor, with `ex.Color.Blue` as the default, and updated test specs ((#846) [https://github.com/excaliburjs/Excalibur/issues/846])
```

Comment:
1. I would move this to under Changed because it's not a "new" feature per se
2. I would rephrase as "Added `backgroundColor` to engine options as part of `Engine` constructor"

No need to mention default color (didn't really change) or test specs (end users don't care)